index.html

```diff
@@ -68,25 +75,471 @@ <h3 class="main-subheading">
+            </li>
+            <li class="company-item">
+              <a href="#" class="nav-link">Creators</a>
+            </li>
+            <li class="company-item">
+              <a href="#" class="nav-link">Blog Articles</a>
+            </li>
+          </ul>
+          <ul class="products-nav">
+            <li class="products-item">
+              <h3 class="products-heading">Products</h3>
+            </li>
+            <li class="products-item">
+              <a href="#" class="nav-link">Website</a>
+            </li>
+            <li class="products-item">
+              <a href="#" class="nav-link">Mobile App</a>
+            </li>
+            <li class="products-item">
+              <a href="#" class="nav-link">Social Media Feeds</a>
+            </li>
+            <li class="products-item">
+              <a href="#" class="nav-link">Presentation</a>
+            </li>
+            <li class="products-item">
+              <a href="#" class="nav-link">Product Mockups</a>
+            </li>
+            <li class="products-item">
+              <a href="#" class="nav-link">Illustrations</a>
+            </li>
+          </ul>
+          <ul class="account-nav">
+            <li class="account-item">
+              <h3 class="account-heading">Account</h3>
+            </li>
+            <li class="account-item">
+              <a href="#" class="nav-link">Your Cart</a>
+            </li>
+            <li class="account-item">
+              <a href="#" class="nav-link">Check Out</a>
+            </li>
+            <li class="account-item">
+              <a href="#" class="nav-link">Refund Policy</a>
+            </li>
+            <li class="account-item">
+              <a href="#" class="nav-link">Company Policy</a>
+            </li>
+            <li class="account-item">
+              <a href="#" class="nav-link">Send A Message</a>
+            </li>
+            <li class="account-item">
+              <a href="#" class="nav-link">Customer Support</a>
+            </li>
+          </ul>
+        </nav>
+        <div class="footer-text-container">
+          <img src="#" alt="#" class="footer-logo" />
+          <p class="footer-text">
+            Your premium digital assets solution with high quality and
+            comprehensive range of products and services.
+          </p>
+          <div class="social-links">
+            <a href="https://x.com/" class="social-link">TW</a>
+            <a href="https://www.instagram.com/" class="social-link">IG</a>
+            <a href="https://www.facebook.com/" class="social-link">FB</a>
+          </div>
+        </div>
+      </div>
+      <div class="footer-end">
+        <div class="footer-end-links">
+          <a href="#" class="privacy-link">Privacy Policy</a>
+          <a href="#" class="terms-link">Terms and Condition</a>
+          <a href="#" class="site-map-link">Site Map</a>
+        </div>
+        <div class="footer-end-copy">
```

Comment:
The `<div>` element here is not the best solution. Div is usually a container where we place data. It can contain other block elements, mainly used for subsequent content styling since the div itself does not carry any meaning. For text content elements, use `<h1>`-`<h6>` or `<p>`. More details on HTML5 content models: https://www.w3.org/TR/2011/WD-html5-20110525/content-models.html